<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Thank you for taking a part of our survey!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="results_btn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(155, 250, 255)</string>
   </property>
   <property name="text">
    <string>View Results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resub_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(183, 255, 156)</string>
   </property>
   <property name="text">
    <string>Resubmit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
